packages/terra-tabs/tests/jest/TabPane.test.jsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { shallowWithIntl } from 'terra-enzyme-intl';
 import Pane from '../../src/TabPane';
 
 describe('TabPane', () => {
   it('should render a default component with label', () => {
-    const wrapper = shallow(<Pane label="Default" />).dive();
+    const wrapper = shallowWithIntl(<Pane label="Default" />).dive();
```

Comment:
Dive has not been added for this file

Reply:
Can you please check this one again? , it is already there .
const wrapper = shallowWithIntl(<Pane label="Default" />).dive();